src/main/java/mate/academy/bookshop/model/CartItem.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.bookshop.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "cart_items")
```

Comment:
do not forget about equals & hashcode & toString, but exclude other entity-fields from it like book or cart